include/net/sntp.h

```diff
@@ -45,15 +51,29 @@ int sntp_init(struct sntp_ctx *ctx, struct sockaddr *addr,
 	      socklen_t addr_len);
 
 /**
- * @brief Send SNTP request
+ * @brief SNTP query with seconds precision (deprecated)
  *
  * @param ctx Address of sntp context.
  * @param timeout Timeout of waiting for sntp response (in milliseconds).
  * @param epoch_time Seconds since 1 January 1970.
  *
  * @return 0 if ok, <0 if error.
  */
-int sntp_request(struct sntp_ctx *ctx, u32_t timeout, u64_t *epoch_time);
+__deprecated int sntp_request(struct sntp_ctx *ctx, u32_t timeout,
+			      u64_t *epoch_time);
+
+/**
+ * @brief Perform SNTP query
+ *
+ * @param ctx Address of sntp context.
+ * @param timeout Timeout of waiting for sntp response (in milliseconds).
+ * @param time Timestamp including integer and fractional seconds since
+ * 1 Jan 1970 (output).
```

Comment:
In NTP timestamps, the seconds value is calculated relative to 0h on 1 January 1900 and not from Unix epoch.

Reply:
This has nothing to do with NTP, nor do I change anything, it was relative to 1 Jan 1970, and remains to be such (there's no issue with that, only POSIX compatibility with `time_t` (ideally, we'd use `time_t` directly, but I'm yet brave enough to make that change, will likely do it later)).